model/permission.go

```diff
@@ -3,17 +3,17 @@
 
 package model
 
+const (
```

Comment:
Guessing this model struct was only used internally?  Just want to make sure we're not breaking v4 APIs.

Reply:
Assuming you are asking about the `Role` struct, it hasn't been removed, just moved to another file. And yes, it wasn't exposed in the public API until now.